phoonnx/util.py

```
@@ -18,6 +18,16 @@
     LOG = logging.getLogger("phoonnx")
 
 
+def normalize_lang(lang: str) -> str:
+    if lang == "tgl" or lang == "tl":
+        # HACK: langcodes erroneously (debatable) changes this one to "fil"
+        return "tl"
+    try:
+        return standardize_tag(lang)
+    except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)